src/main/java/com/google/longrunning/OperationsSettings.java

```diff
@@ -206,7 +206,7 @@ public ListOperationsPagedResponse createPagedListResponse(
                 UnaryCallable<ListOperationsRequest, ListOperationsResponse> callable,
                 ListOperationsRequest request,
                 CallContext context) {
-              return new ListOperationsPagedResponse(
+              return ListOperationsPagedResponse.callApiAndCreate(
```

Comment:
It looks like we have unrolled one level and uncovered the same problem here that existed previously in the response constructor: the function name here (`createPagedListResponse`) only implies simple creation, when in fact the API is called first. It seems like this method should be called `callAndCreateResponse` or something like that.